swig/python2/tests/NotificationTest.py

```diff
@@ -417,7 +417,7 @@ def test_notify_same_path(self):
          ["DELETED", "/ietf-interfaces:interfaces/interface[name='eth0']/ietf-ip:ipv4/address[ip='192.168.2.100']/ip"],
          ["DELETED", "/ietf-interfaces:interfaces/interface[name='eth0']/ietf-ip:ipv4/address[ip='192.168.2.100']/prefix-length"],
          ["DELETED", "/ietf-interfaces:interfaces/interface[name='eth0']/ietf-ip:ipv4/enabled"],
-         # ["DELETED", "/ietf-interfaces:interfaces/interface[name='eth0']/ietf-ip:ipv4/mtu"],
```

Comment:
Hi Antonio,
the issue is that these [two lines](https://github.com/sysrepo/sysrepo/blob/8a07d3073be37b80ff6166335d4e691a46170087/tests/python/TestModule.py#L169-L170) got lost during refactoring. Adding them back should resolve the issue.
Regards,
Lukas